src/main/resources/META-INF/plugin.xml

```diff
@@ -382,6 +382,17 @@
             implementationClass="com.jetbrains.snakecharm.inspections.SmkPyUnboundLocalVariableInspection"
     />
 
+    <localInspection
+            language="Snakemake" shortName="SmkRedundantComaInspection"
+            enabledByDefault="true"
+            level="WARNING"
+            suppressId="SmkRedundantComaInspection"
```

Comment:
Please remove "Inspection" suffix from `suppressId` all other inspections don't have such suffix.

Reply:
Fixed

Reply:
`Comma` should be with double `m`, please fix all affected names